lib/constants/category.ts

```diff
@@ -0,0 +1,24 @@
+export type Category =
+  | 'ansible'
+  | 'batect'
+  | 'bazel'
+  | 'c'
+  | 'cd'
+  | 'ci'
+  | 'dart'
+  | 'docker'
+  | 'dotnet'
+  | 'elixir'
+  | 'golang'
```

Comment:
```suggestion
  | 'go'
```

https://go.dev/doc/faq#go_or_golang

Reply:
`golang` is used in `languages`. IMO as we keep `js` for compatibility, we should follow the same with `golang`.  

We can introduce this while removing languages if you are feeling strongly about this.